pipeline/install-prokka.sh

```diff
@@ -6,41 +6,41 @@ cd /mnt
 curl -O ftp://selab.janelia.org/pub/software/hmmer3/3.1b1/hmmer-3.1b1.tar.gz
 tar xzf hmmer-3.1b1.tar.gz
 cd hmmer-3.1b1/
-./configure --prefix=/usr && make && make install
+./configure --prefix=${HOME} && make && make install
 
 cd /mnt
 curl -O http://mbio-serv2.mbioekol.lu.se/ARAGORN/Downloads/aragorn1.2.36.tgz
 tar -xvzf aragorn1.2.36.tgz
 cd aragorn1.2.36/
 gcc -O3 -ffast-math -finline-functions -o aragorn aragorn1.2.36.c
-cp aragorn /usr/local/bin
+cp aragorn ${HOME}/bin/
 
 cd /mnt
 curl -O http://prodigal.googlecode.com/files/prodigal.v2_60.tar.gz
 tar xzf prodigal.v2_60.tar.gz
 cd prodigal.v2_60/
 make
-cp prodigal /usr/local/bin
+cp prodigal ${HOME}/bin/
 
 cd /mnt
 curl -O ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/linux64.tbl2asn.gz
 gunzip linux64.tbl2asn.gz
 mv linux64.tbl2asn tbl2asn
 chmod +x tbl2asn
```

Comment:
Did you try it? How does it halt? Anyone is allowed to chmod files that they own.